05f-choosing_histology_tools.Rmd

```diff
@@ -0,0 +1,14 @@
+
+# Considerations for choosing histology tools
+
+## Learning Objectives
+
+```{r, fig.alt = "", out.width = "100%", echo = FALSE}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1ME0NbcIBmnHJRhX3JJyCwJuuomkl_BjJp6lD5oD5WnU/edit#slide=id.gd422c5de97_0_10")
+```
+
+## Tools for image standardization
+
+## Why would you use one tool over the other?
+
+## What parameters, or caveats should be known about each tool?
```

Comment:
Same comment as before with the parallel radiology section

Reply:
Because we are just getting the outline draft in here I think its okay if is changed later.